### PR TITLE
chore(cdk): polymorpheus migrations

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/cdk/schematics/utils/templates/elements.ts [hunk 1]
```diff
@@ -3,10 +3,9 @@ import {ChildNode, Element, parseFragment} from 'parse5';
 const ALWAYS_TRUE_HANDLER = (): true => true;
 
 export function findElementByFn(
```
Comment (nsbarsukov) [inline projects/cdk/schematics/utils/templates/elements.ts:5]: let rename it =)

```suggestion
export function findElementsByFn(
```

File: projects/cdk/schematics/utils/templates/elements.ts [hunk 2]
```diff
@@ -23,13 +22,21 @@ export function findElementByFn(
         });
     };
 
-    visitNodes(document.childNodes);
+    visitNodes(nodes);
 
     return elements;
 }
 
+export function findElementInTemplateByFn(
```
Comment (nsbarsukov) [inline projects/cdk/schematics/utils/templates/elements.ts:30]: and here too

```suggestion
export function findElementsInTemplateByFn(
```

File: projects/cdk/schematics/ng-update/tests/schematic-migrate-polymorpheus.spec.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            class="t-card"
+            [src]="icon"
+        ></tui-svg>
+    </ng-template>
+</div>
+`;
+
+const TEMPLATE_AFTER = `
+<div
+            >
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/tests/schematic-migrate-polymorpheus.spec.ts:56]: it is still valid HTML, isn't it ?

**Review by nsbarsukov — APPROVED** [review]

**Review by splincode — APPROVED** [review]

File: projects/cdk/schematics/utils/templates/elements.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/cdk/schematics/utils/templates/elements.ts]: good idea, done ✅

**Pull request merged**